packages/cfd/src/features/types/containers.types.ts

```diff
@@ -0,0 +1,375 @@
+import React from 'react';
+import {
+    GetSettings,
+    ResidenceList,
+    GetAccountStatus,
+    VerifyEmailResponse,
+    DetailsOfEachMT5Loginid,
+} from '@deriv/api-types';
+import { FormikHelpers as FormikActions } from 'formik';
+import {
+    TExistingData,
+    TClickableDescription,
+    TJurisdictionCardItems,
+    TJurisdictionCardSection,
+    TJurisdictionCardItemVerification,
+    TJurisdictionCardSectionTitleIndicators,
+    TModifiedTradingPlatformAvailableAccount,
+} from './components.types';
+import RootStore from 'Stores/index';
+import { TCFDPlatform, TTradingPlatformAvailableAccount, TTokens, TAccountCategory, TMarketType } from './shared.types';
+
+type TCFDPasswordFormValues = { password: string };
+
+export type TCFDPersonalDetailsContainerProps = {
+    onSubmit: (index: number, value: { [key: string]: string }) => void;
+};
+
+// cfd-change-password-confirmation
+export type TCFDChangePasswordConfirmationProps = {
+    platform: TCFDPlatform;
+    className?: string;
+    context?: RootStore;
+    onCancel: () => void;
+    confirm_label?: string;
+    onConfirm: (values: TCFDPasswordFormValues, actions: FormikActions<TCFDPasswordFormValues>) => void;
+};
+
+export type TCFDDashboardContainer = {
+    active_index: number;
+    platform: TCFDPlatform;
```

Comment:
I see that we are repeating this many times around. Maybe we can have a type for platform and extend that to every other type that needs it.

Reply:
We're already using `TCFDPlatform` as a shared type wherever it is needed. What do you suggest ?